src/components/questions/IndividualQuestion.jsx

```diff
@@ -24,7 +19,7 @@ const Question = (props) => {
           </div>
         </div>
         <div className={styles.answerRow}>
-          <AnswerList answers={answers.results} questionid={props.questionid}/>
+          <AnswerList answers={answers.results} questionid={props.question_id}/>
```

Comment:
This file was changed during debugging with Yosef for the error happened on changing the product id of context
@ygroener  would you please double check these code ? Thanks!